client/src/components/form/UpdateBiodata.tsx

```
@@ -1,160 +1,188 @@
-              <button onClick={handleReset} className="formbtn">
-                Reset
-              </button>
-            </Box>
-          </>
-        ) : (
-          <>
+    <div className="p-4">
+      {isSubmitted ? (
+        <div>
+          <h2>Form Submitted Successfully!</h2>
+          <button onClick={handleReset}>Reset Form</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)